blob/drivertest/drivertest.go

```diff
@@ -2087,24 +2094,30 @@ func testSignedURL(t *testing.T, newHarness HarnessMaker) {
 	// PUT the blob. Try with all URLs, only putURL should work.
```

Comment:
Can you extend this comment to say "only putURL should work, and only when the content type matches" or something?

Or alternatively, put comments by each testcase explaining why they fail.